client/js/store/modules/auth.js

```diff
@@ -52,11 +52,11 @@ export default {
     },
   },
   actions: {
-    requestLogin(context, { username, password }) {
+    async requestLogin(context, { username, password }) {
       return handleLogin(context, username, password);
     },
 
-    requestSignup(context, { username, password }) {
+    async requestSignup(context, { username, password }) {
```

Comment:
These need to be async so cypress can properly await them when dispatching them from the window.